app/assets/stylesheets/application.tailwind.css

```diff
@@ -1,9 +1,9 @@
+@import 'actiontext.css';
```

Comment:
This would make it so that tailwind preflight styles will override these. Is this what we want?